contracts.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Contracts API
+ * @module contracts
+ */
+
+const abis = require('./resources/abis.json');
+
+/**
+ * Create a collection of Urbit contracts, given a web3 instance and their
+ * provided addresses.
+ * @param {Object} web3 - A web3 instance.
+ * @param {Object} ship - An addresses object.  Must provide addresses for
+ *   constitution, ships, polls, and pool contracts, at those respective key
+ *   names.
+ * @return {Object} The initialised contracts.
+ */
+function initContracts(web3, addresses) {
```

Comment:
It might have just fallen out in the refactor.  But in general I've probably been thinking of this library in network-generic fashion, so that clients must always supply their own addresses.  Just thinking in terms of (abstract) dependencies: I'd like to pin a version of the *contracts* as a dependency, rather than a particular network.

(This may be overly general on my part -- there is only one Ethereum mainnet, after all. 😁)

Reply:
Not only is there just one Ethereum mainnet, there is (or, will be) exactly one collection of contracts we call the Constitution. Of course for testnet cases you still want to be able to specify manually. I'm fine with pinning for the real-world case though.